pom.xml

```diff
@@ -181,9 +181,9 @@
       <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
     </repository>
     <snapshotRepository>
-      <id>gforge.inria.fr-snapshot</id>
-      <name>inria-snapshots</name>
-      <url>scp://scm.gforge.inria.fr/home/groups/spoon/htdocs/repositories/snapshots</url>
+      <id>maven.inria.fr-snapshot</id>
+      <name>http://maven.inria.fr-snapshots</name>
```

Comment:
What about avoiding `http://`, it may be confusing, looking like a URL and not a name.

Reply:
Ok I'll change it.